transfer.js

```diff
@@ -0,0 +1,63 @@
+const transferForm = document.getElementById("transfer-form");
+const selectBeneficiaryAccountNumberElem = document.getElementById(
+  "beneficiary-account-number"
+);
+const transferAmountElem = document.getElementById("transfer-amount");
+const accountPinElem = document.getElementById("account-pin");
+const currentUser = getUserByAccountNumber(currentUserAccountNumber);
+const allUsers = getAllUsers();
+// populate users in our database into the select field and remove the current user account number
+allUsers.forEach((user) => {
```

Comment:
Suggested:

```
allUsers.
    .filter((user) => user.accountNumber !== currentUserAccountNumber)
    .forEach((user) => {
      const optionElem = document.createElement("option");
      optionElem.value = user.accountNumber;
      optionElem.textContent = `${user.accountNumber}`;
      selectBeneficiaryAccountNumberElem.append(optionElem);
    });
```

`allUsers.filter` will filter out the current user object from the `allUsers` array, leaving you with the rest of the users to apply `.forEach` on.

Reply:
Done